src/SUMMARY.md

```diff
@@ -8,6 +8,9 @@
 - [Preface](overview/preface.md)
 - [Roles](overview/roles.md)
 - [Requirements](overview/requirements.md)
+- [Group_setups]()
+    - [Creating and Using an FTP Setup](subtitling/group setups/ftp.md)
+    - [Creating and Using a GitHub Setup](subtitling/group setups/github.md)
```

Comment:
Spaces in folder names are usually a bad idea and it appears mdbook doesn't like them either.

Furthermore, you used differing casing in the actual folder name than how you reference them here.